app/controllers/admin/api/api_docs_services_controller.rb

```diff
@@ -105,7 +109,7 @@ def destroy
   protected
 
   def api_docs_params(*extra_params)
-    permit_params = %i(name body description published skip_swagger_validations) + extra_params
+    permit_params = %i[name body description published skip_swagger_validations service_id] + extra_params
```

Comment:
Thee service should exist and belong to the provider right?

Reply:
yes 😄The _belong to the provider_ is being checked in a validation (second commit). The _existence_ should be checked by the foreign key, right?

Reply:
The foreign key is too late, it will through a database error which will lead to 500 to the user

Reply:
Thinking about it... if it doesn't exist, it means it does not belong to the account either, so it would already return a proper error about it. But this case requires a test then 👍 and I will see if this theory is valid 😄